app/views/components/_share-links.html.erb

```diff
@@ -0,0 +1,18 @@
+<% title ||= 'Share this page' %>
+
+<div class="app-c-share-links-wrapper">
+  <h2><%= title %></h2>
+  <div class="app-c-share-links" data-module="track-share-button-clicks">
+    <%= link_to share_urls[:facebook],
```

Comment:
I think I'd prefer `facebook_href` to `share_urls[:facebook]` – ie an API similar to our other link components